### ISSUE TITLE
Contact Form does not work

### ISSUE THREAD

**Issue by tisto** [plone/plone.de#37]
I just realized that the contact form is not working:

https://plone.de/contact-form

![Kontaktformular](https://github.com/plone/plone.de/assets/632105/a3e58291-465c-4830-b0af-b56ccfae4c62)

**Comment by tisto** [comment]
@fredvd @ericof do we have a Plone.org mailserver that we can use to send emails?

**Comment by fredvd** [comment]
@tisto Yes we do, we have a mailgun account that we connect. I'll check and update the config on plone.de

Hmmm.... but then mailgun should be authorative for plone.de if you want to send e-mails from that name. Might be a bit trickier than just adding the account.... writing/thinking out loud, no it's fine, the plone.de DNS servers are also under 'our' control on the cloudflare DNS.  We can add mailgun tot the spf record.

**Comment by acsr** [comment]
Still Fehler on Send: Unsuccessful HTTP response.
I need this to use it in my Success Story Editors contact options!
https://plone.de/vsp is hidden content for now. I want to mention this form at the end
And: How are these mails processed? 
**I step up to be in the mailing list!**

**Comment by acsr** [comment]
By the way a monthly "I am alive" mail from the Plone sites mailsystem would help to check if the mail workflow is up and running. This could be an option in the standard Plone mailhost management. The subject and body should be customizable and an option to send some dynamically created summary of the state of the system would be great as well.

**Comment by fredvd** [comment]
Sorry, I’ll pick this up asap. I noted it as a todo at the beginning of the spriint, but didn’t focus on plone.de ar all

**Comment by tisto** [comment]
@fredvd @acsr status: still not working.

**Comment by fredvd** [comment]
@tisto @acsr My apologies for the long delay. Fixed.  I have configured the mail control panel to use our Plone Foundation mailgun account. 

So... where should the messages go to? If we want more than one recipient and I'll assume we do, I'll have to create a plonede@plone.org maillinglist to which we can subscribe the final recipients and forward incoming messages through the mailgun plone domain (mg.plone.org)

**Comment by acsr** [comment]
THX, @fredvd when creating the [plonede@plone.org](mailto:plonede@plone.org) maillinglist, can we make sure that the sender mail address contains/has something explicit/obvious like noreply.site.plonede@plone.org to recognize the origin when working with other workflows in the same list. plonede@plone.org as sender is for me too generic. Is there anything required by the mailgun account? Like using a particular top level domain? (I can see the mg subdomain there)

**Comment by TimoBroeskamp** [comment]
Since this is fixed this one can be closed too.

**Comment by tisto** [comment]
@fredvd @acsr I am currently not on the plonede@plone.org mailinglist. Can one of you please add me?

**Comment by acsr** [comment]
@fredvd I never got at least an announcement mail that a list plonede@plone.org exists or a mail from the sender [noreply.site.plonede@plone.org](mailto:noreply.site.plonede@plone.org) at all. @tisto can you check your inmails as well?
